<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAirplaneDialog</class>
 <widget class="QDialog" name="AddAirplaneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 10pt;
}

QDialog {
	
	background-color: rgb(0, 0, 0);
}

QLabel {
	color: rgb(255, 246, 0);
}

QLineEdit, QComboBox, QSpinBox {
	color: rgb(255, 246, 0);
	
	background-color: rgb(0, 0, 0);
	border: 1px solid #fff600;
}

QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(255, 246, 0);
	border: 3px solid #fff600;
}

QPushButton:hover {
	background-color: #fff600;
	color: black;
}

#frame {
	
	background-color: rgb(255, 255, 255, 80);
	border-radius: 4px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblAirline">
        <property name="text">
         <string>Airplane:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAirline"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblSerial">
        <property name="text">
         <string>Serial:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSerial"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblNumOfSeats">
        <property name="text">
         <string>Number Of Seats:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnNumOfSeats"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btnSubmit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
